runHarness/DataManagers/AuctionKubernetesDataManager.pm

```diff
@@ -248,31 +251,31 @@ sub pretouchData {
 	my $appInstanceNum = $self->appInstance->instanceNum;
 	$logger->debug( "pretouchData for workload {}, appInstance {}", $workloadNum, $appInstanceNum);
 	
-#	my $cluster = $self->host;
-#	my $namespace = $self->appInstance->namespace;
-#	
-#	my $logName = "$logPath/PretouchData_W${workloadNum}I${appInstanceNum}.log";
-#	my $logHandle;
-#	open( $logHandle, ">$logName" ) or do {
-#		$console_logger->error("Error opening $logName:$!");
-#		return 0;
-#	};
-#	print $logHandle "Exec-ing /pretouchData.sh\n";
-#	$logger->debug("Exec-ing /pretouchData.sh in cassandra-0");
-#	my ($cmdFailed, $outString) = $cluster->kubernetesExecOne("cassandra", "/pretouchData.sh", $namespace);
-#	if ($cmdFailed) {
-#		$logger->debug( "Pretouching data failed for workload $workloadNum, appInstance $appInstanceNum. \$cmdFailed = $cmdFailed" );
-#		print $logHandle "Pretouching data failed for workload $workloadNum, appInstance $appInstanceNum. \$cmdFailed = $cmdFailed\n";
-#		return 0;
-#	}
-#	else {
-#		$logger->debug( "Pretouching data failed for workload $workloadNum, appInstance $appInstanceNum. \$outString = $outString" );
-#		print $logHandle "Pretouching data failed for workload $workloadNum, appInstance $appInstanceNum. \$outString = $outString\n";
-#		return 1;
-#	}
-#	$logger->debug( "pretouchData complete for workload ", $workloadNum );
-#	close $logHandle;
-#	return 1;
+	my $cluster = $self->host;
+	my $namespace = $self->appInstance->namespace;
+	
+	my $logName = "$logPath/PretouchData_W${workloadNum}I${appInstanceNum}.log";
+	my $logHandle;
+	open( $logHandle, ">$logName" ) or do {
+		$console_logger->error("Error opening $logName:$!");
+		return 0;
+	};
+	print $logHandle "Exec-ing /pretouchData.sh\n";
+	$logger->debug("Exec-ing /pretouchData.sh in cassandra-0");
+	my ($cmdFailed, $outString) = $cluster->kubernetesExecOne("cassandra", "/pretouchData.sh", $namespace);
+	if ($cmdFailed) {
+		$logger->debug( "Pretouching data failed for workload $workloadNum, appInstance $appInstanceNum. \$cmdFailed = $cmdFailed" );
+		print $logHandle "Pretouching data failed for workload $workloadNum, appInstance $appInstanceNum. \$cmdFailed = $cmdFailed\n";
+		return 0;
+	}
+	else {
```

Comment:
Is this else block supposed to exist like this? logging "failed"?
Also, the code below is unreachable. (and it does not print $logHandle).

Reply:
The log messages are a copy/past issue, but that is moot because the entire pretouchData path is not used.  This took a long time to execute and didn't improve performance.  I intended to take this all out and forgot.  I will do that now.